pkg/statistics/handle/syncload/stats_syncload_test.go

```diff
@@ -312,11 +323,11 @@ func TestRetry(t *testing.T) {
 	result, err1 := h.HandleOneTask(testKit.Session().(sessionctx.Context), task1, exitCh)
 	require.NoError(t, err1)
 	require.Nil(t, result)
-	select {
```

Comment:
We cannot use the old way to check. the item in the channel will be fetched from ```globalSingleFlight```.